docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+services:
+  api:
+    container_name: backend
```

Comment:
Dont use container_name in stack based deployment

Reply:
Noted, Have made the change